test/recorder.js

```diff
@@ -264,16 +270,51 @@ describe('Recorder', function(){
     return rec.start().then(function(){
       rec.stop();
       expect(rec.state).to.equal('inactive');
-      expect(rec.monitorNode.disconnect).to.have.been.calledOnce;
+      expect(rec.monitorNode.disconnect).to.have.been.calledTwice;
```

Comment:
I won't accept this pull request with this. Disconnect should only be called once.